Docker.DotNet/Endpoints/ImageOperations.cs

```diff
@@ -51,7 +51,7 @@ internal ImageOperations(DockerClient client)
             }
 
             string path = string.Format(CultureInfo.InvariantCulture, "images/{0}/json", name);
-            DockerApiResponse response = await this.Client.MakeRequestAsync(new[] {NoSuchImageHandler}, HttpMethod.Get, path, null).ConfigureAwait(false);
+            DockerApiResponse response = await this.Client.MakeRequestAsync(new[] { NoSuchImageHandler }, HttpMethod.Get, path, null).ConfigureAwait(false);
```

Comment:
Kind of... this is the spacing that gets inserted by VSCode when you use their auto-format for C# code, so anytime anyone edits this file and uses auto-formatting, it will just end up getting inserted again.